stock_picking_line_sequence/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA - Damien Crier, Alexandre Fayolle
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock picking lines with sequence number',
+    'summary': 'Manages the order of stock moves by displaying its sequence',
+    'version': '8.0.1.0.0',
+    'category': 'Warehouse Management',
+    'author': "Camptocamp, "
+              "Eficent, "
+              "Serpent CS, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.camptocamp.com',
```

Comment:
URL should be `https://github.com/OCA/stock-logistics-workflow`

Reply:
corrected